packages/core/.build/scripts/auto-export-all-components.ts

```
@@ -10,20 +9,77 @@ interface ComponentInfo {
 
-  // 扫描组件目录
   if (await fs.exists(componentsDir)) {
     const dirs = await fs.readdir(componentsDir);
 
     for (const dir of dirs) {
       const compPath = path.join(componentsDir, dir, 'index.vue');
       if (await fs.exists(compPath)) {
         const compName = dir.replace(/(^\w|-\w)/g, (m: string) =>
-          m.replace('-', '').toUpperCase());
-
+          m.replace('-', '').toUpperCase()
```

ESLint

[error] Should not have line breaks between items, in node CallExpression

(antfu/consistent-list-newline)